package.json

```diff
@@ -136,6 +137,7 @@
     "redux": "3.0.4",
     "redux-form": "7.0.2",
     "redux-thunk": "1.0.0",
+    "reselect": "3.0.1",
```

Comment:
Yes, I've added a bunch of `TODO`s to change its usages to Calypso's `createSelector`, but I didn't want to hold on to those changes any longer, since there are 3 people working on this.